pkg/client/cache/delta_fifo.go

```diff
@@ -452,6 +452,18 @@ func (f *DeltaFIFO) Replace(list []interface{}, resourceVersion string) error {
 	return nil
 }
 
+// Resync will send a sync event for each item
+func (f *DeltaFIFO) Resync() error {
+	f.lock.RLock()
+	defer f.lock.RUnlock()
+	for _, item := range f.items {
+		if err := f.queueActionLocked(Sync, item); err != nil {
+			return fmt.Errorf("couldn't sync object: %v", err)
```

Comment:
s/sync/queue/
[This is just queueing objects, not really performing the sync operation]